app/models/consumer.rb

```diff
@@ -0,0 +1,6 @@
+class Consumer < ApplicationRecord
```

Comment:
Shouldn't we call it Customer?

Reply:
That's what I had in mind but it appears the others chose to call it this way in the ERD diagram, we'll see what their idea on the subject is tomorrow morning

Reply:
I think we decided Customer? I think either one is OK we just need to use the same everywhere

Reply:
Yes, I remember of Customer, but no problem to change if we all agreed